### PR TITLE
Update python samples sdk 2.5

### PR DESCRIPTION
**Pull request by ajlandau**
Updated the samples to reference Zivid Two for the sample data
Updated samples to use enum for complete settings HDR
Updated links for tutorial

### CODE REVIEW

File: source/camera/advanced/capture_hdr_loop.py [hunk 1]
```diff
@@ -19,7 +19,10 @@ def _main():
     camera = app.connect_camera()
 
     for i in range(1, 4):
-        settings_file = Path() / get_sample_data_path() / f"Settings/Zivid One/Settings0{i :01d}.yml"
+        camera_model = camera.info.model_name
+        camera_path = "Settings/" + camera_model[0:9] + f"/Settings0{i :01d}.yml"
+        settings_file = Path() / get_sample_data_path() / camera_path
```
Comment (SatjaSivcev) [inline source/camera/advanced/capture_hdr_loop.py:24]: In C++ and C# these two are one line. I suggest to make them one line here as well.

https://github.com/zivid/zivid-cpp-samples/blob/d092e7bfe41aec7cd7a14281e28a3b04f9c4a383/source/Camera/Basic/CaptureWithSettingsFromYML/CaptureWithSettingsFromYML.cpp#L20-L23

https://github.com/zivid/zivid-csharp-samples/blob/ff844b4307451c61fba6ca92b279a0d9ceb0defc/source/Camera/Basic/CaptureWithSettingsFromYML/CaptureWithSettingsFromYML.cs#L20-L24

File: source/camera/basic/capture_with_settings_from_yml.py [hunk 2]
```diff
@@ -16,7 +16,12 @@ def _main():
     print("Connecting to camera")
     camera = app.connect_camera()
 
-    settings_file = Path() / get_sample_data_path() / "Settings/Zivid One/Settings01.yml"
+    print("Creating settings from file")
+
+    camera_model = camera.info.model_name
+    camera_path = "Settings/" + camera_model[0:9] + "/Settings01.yml"
+    settings_file = Path() / get_sample_data_path() / camera_path
```
Comment (SatjaSivcev) [inline source/camera/basic/capture_with_settings_from_yml.py:23]: Same as https://github.com/zivid/zivid-python-samples/pull/123/commits/41ed612977efe77cae9d84e17e741c23016b8dee#r740578146

File: source/camera/basic/capture_hdr_complete_settings.py [hunk 3]
```diff
@@ -17,28 +17,59 @@ def _main():
...
-            ),
-            zivid.Settings.Acquisition(
-                aperture=4.0,
-                exposure_time=datetime.timedelta(microseconds=40000),
-                brightness=1.8,
-                gain=2.0,
-            ),
-        ],
-    )
+    if (
```
Comment (SatjaSivcev) [inline source/camera/basic/capture_hdr_complete_settings.py:20]: In C++ and C# this is a function called getExposures, maybe try to implement the same here.


https://github.com/zivid/zivid-cpp-samples/blob/d092e7bfe41aec7cd7a14281e28a3b04f9c4a383/source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cpp#L15-L35

**Review by SatjaSivcev — CHANGES_REQUESTED** [review]
Some minor comments. Fix them and make commit messages nicer (capital letters and right tense)

File: source/camera/advanced/capture_hdr_loop.py (hunk 1 above)
Comment (ajlandau) [reply source/camera/advanced/capture_hdr_loop.py:24]: I tried, it gets mad about how the path is made and throws errors.

Comment (ajlandau) [reply source/camera/advanced/capture_hdr_loop.py:24]: So it did not work to make them the exact same.

Comment (SatjaSivcev) [reply source/camera/advanced/capture_hdr_loop.py:24]: Have you tried this:

`Path() / get_sample_data_path() / camera_path / Path("Settings/" + camera_model[0:9] + f"/Settings0{i :01d}.yml")`

Comment (ajlandau) [reply source/camera/advanced/capture_hdr_loop.py:24]: Fixed

File: source/camera/basic/capture_with_settings_from_yml.py (hunk 2 above)
Comment (ajlandau) [reply source/camera/basic/capture_with_settings_from_yml.py:23]: Fixed

File: source/camera/basic/capture_hdr_complete_settings.py (hunk 3 above)
Comment (ajlandau) [reply source/camera/basic/capture_hdr_complete_settings.py:20]: I think this is just a place were we will differ for pythonic vs C++
I do not think this needs to be made into a function in this case.

Comment (ajlandau) [reply source/camera/basic/capture_hdr_complete_settings.py:20]: I can make the print statements match though if you want.

**Review by SatjaSivcev — APPROVED** [review]
LGTM

**Pull request merged**